### DISCUSSION TITLE
[FE] 디자인 토큰 및 기본 컴포넌트 리팩토링

### DISCUSSION THREAD

**Discussion by Hain-tain** [woowacourse-teams/2024-code-zap#243] [결정 사항]
1. 디자인 토큰 적용 실효성에 대한 이야기 (지금 사용하지 않는 컬러들도 강박적으로 다 넣어두었는데, 진짜 다 쓰지는 않는 것 같다. 사용하는 컬러들만 뽑아서 넣을지, 그럼 어떠어떠한 걸 사용할건지 정해두면 좋을 것 같다…!)
3. 버튼 컴포넌트 리펙토링 방향 및 버튼 컴포넌트로 어디까지 대응해줄 것인가에 대한 이야기
4. hook 폴더링 어떻게 하는게 좋을까?

**Comment by Jaymyong66** [comment]
사실상 컬러 팔레트를 전부 사용하지 않는다.

**Comment by Hain-tain** [comment]
1. 디자인 토큰 적용 실효성에 대한 이야기
- 피그마 디자인시스템 페이지에 각자 자신이 사용하고 있는 컬러 오른쪽에 빼고, 사용처 또는 사용 용도 적어두기!
